packages/forma-36-react-components/src/components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import cn from 'classnames';
+import Heading from '../Typography/Heading';
+import Paragraph from '../Typography/Paragraph';
+
+const styles = require('./EmptyState.css');
+
+export type EmptyStateProps = {
+  /**
+   * Class names to be appended to the className prop of the component
+   */
+  className?: string;
+  /**
+   * Child nodes to be rendered in the component
+   */
+  children?: React.ReactNode;
+  /**
+   * An ID used for testing purposes applied as a data attribute (data-test-id)
+   */
+  testId?: string;
+  /**
+   * Props for image block
+   */
+  image?: ImageProps;
+  /**
+   * Heading text
+   */
+  heading: string;
+  /**
+   * Description text
+   */
+  description: string;
+} & typeof defaultProps;
+
+interface ImageProps {
+  /**
+   * Url of the illustration for empty state
+   */
+  url: string;
+  /**
+   * Width of the illustration for empty state
```

Comment:
Maybe update comment with an example value to make it clear that `width`/`height` expects the unit too:

```
Width of the illustration for empty state including unit (e.g. '250px')
```